packages/integrations/package.json

```diff
@@ -18,6 +18,7 @@
   "dependencies": {
     "@sentry/types": "5.20.1",
     "@sentry/utils": "5.20.1",
```

Comment:
```suggestion
    "@sentry/types": "5.21.3",
    "@sentry/utils": "5.21.3",
```

Reply:
It looks like those deps were just updated to `5.21.4`, so I used that version instead!